### PR TITLE
Fix more misspellings

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
_Implementor_ is not exactly a misspelling, but:
* From [Garner's Modern English Usage (4 ed.)](https://www.oxfordreference.com/display/10.1093/acref/9780190491482.001.0001/acref-9780190491482-e-4046):
  > Although the variant spelling ✳implementor predominated for much of the late 20th century, today implementer is considered standard.
* The Google Ngram Viewer shows a [ratio of almost 10:1 in 2019](https://books.google.com/ngrams/graph?content=implementor%2Cimplementer&year_start=1900&year_end=2019&corpus=en-2019).

### CODE REVIEW

File: vectors/cryptography_vectors/asymmetric/RSA/FIPS_186-2/Readme.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@ Example test files for FIPS 186-2 RSA
 
 2. The file SigGenRSA_186-2.txt contains values for X9.31RSA signature generation with the additional value d added to the file for testing purposes.
 
-3.  The file SigGen15_186-2.txt contains values for RSA PKCS#1 Ver 1.5 signature geenration with the additional value d added to the file for testing purposes.
+3.  The file SigGen15_186-2.txt contains values for RSA PKCS#1 Ver 1.5 signature generation with the additional value d added to the file for testing purposes.
```
Comment (alex) [inline vectors/cryptography_vectors/asymmetric/RSA/FIPS_186-2/Readme.txt:7]: This file is an exact replica of a vector from NIST and shouldn't be modified.

Comment (DimitriPapadopoulos) [reply vectors/cryptography_vectors/asymmetric/RSA/FIPS_186-2/Readme.txt]: Fixed.

File: docs/spelling_wordlist.txt [hunk 2]
```diff
@@ -57,12 +57,12 @@ fallback
 Fernet
 fernet
 FIPS
+geenration
```
Comment (reaperhulk) [inline docs/spelling_wordlist.txt:60]: Why was this added?

Comment (DimitriPapadopoulos) [reply docs/spelling_wordlist.txt:60]: In a (non-sensical) attempt to avoid future spelling errors, but the list of misspellings applies only to the docs.

**Review by reaperhulk — APPROVED** [review]
